src/redux/reducers/bookReducer/book.reducer.js

```diff
@@ -0,0 +1,28 @@
+const initialState = {
+  books: [
+    {
+      genre: 'Romance',
+      title: 'Mike the BBC Reporter',
+      author: 'Michael Garyssan',
+      completed: 64,
+      chapter: 'Chapter 17',
+    },
+    {
+      genre: 'Fiction',
+      title: 'The Alchemist',
+      author: 'Paulo Coelho',
+      completed: 88,
+      chapter: 'Chapter 7',
+    },
+  ],
```

Comment:
- [ ] this is good but you need this one in your components and then call the prop in your booklist